src/renderer/common/apiAction.ts

```diff
@@ -15,58 +15,59 @@ import { v4 as uuidv4 } from "uuid";
-                    requestId,
-                    moduleId,
-                    methodId,
-                    requestData,
-                ),
-            );
+                store.dispatch(
+                    apiActions.request.build(
+                        requestId,
+                        moduleId,
+                        methodId,
+                        requestData,
+                    ),
+                );
 
-            const promise = new Promise<ReturnPromiseType<TApiMethod[T]>>((resolveSubscribe, rejectSubscribe) => {
-                storeUnsubscribe = store.subscribe(() => {
-                    const state = store.getState();
-                    const lastTime = (state.api[requestId]?.lastTime) || 0;
+                const promise = new Promise<ReturnPromiseType<TApiMethod[T]>>((resolveSubscribe, rejectSubscribe) => {
+                    storeUnsubscribe = store.subscribe(() => {
+                        const state = store.getState();
+                        const lastTime = (state.api[requestId]?.lastTime) || 0;
 
-                    if (state.api[requestId]?.data.time > lastTime) {
-                        const data = { ...state.api[requestId].data };
-                        store.dispatch(apiActions.clean.build(requestId));
-                        if (data.error) {
-                            rejectSubscribe(data.errorMessage);
-                            return;
+                        if (state.api[requestId]?.data.time > lastTime) {
+                            const data = { ...state.api[requestId].data };
+                            store.dispatch(apiActions.clean.build(requestId));
+                            if (data.error) {
+                                rejectSubscribe(data.errorMessage);
+                                return;
+                            }
+                            resolveSubscribe(data.result);
                         }
-                        resolveSubscribe(data.result);
-                    }
-                    // handle promise<void>
-                    timeoutId = window.setTimeout(() => {
-                        timeoutId = undefined;
-                        rejectSubscribe("API Timeout");
-                    }, 5000);
+                        // handle promise<void>
+                        timeoutId = window.setTimeout(() => {
+                            timeoutId = undefined;
+                            rejectSubscribe("API Timeout");
+                        }, 5000);
+                    });
                 });
-            });
 
-            promise.then((result) => {
-                resolve(result);
-            }).catch((error) => {
-                reject(error);
-            }).finally(() => {
-                if (storeUnsubscribe) {
-                    storeUnsubscribe();
-                }
-                if (timeoutId) {
-                    clearTimeout(timeoutId);
-                }
+                await promise.then((result) => {
```

Comment:
Ah, I think this is the culprit for the regression bug. Fixing this now...

Reply:
Well, this needs fixing (which I did, and I will push the change), but the problem is elsewhere...

Reply:
Ok, problem solved. Pushing fix now...